internal/db/id.go

```diff
@@ -17,12 +17,13 @@ func NewPublicId(prefix string) (string, error) {
 }
 
 func newId(prefix string) (string, error) {
+	const op = errors.Op("db.newId")
```

Comment:
I think you can just use `const op = "db.newId"`  The `errors.Op` isn't needed and you won't have to cast it when using it later.